docs/api/server/Environment-Variables.md

```diff
@@ -2,6 +2,461 @@
 
 # Environment Variables
 
-Documentation Forthcoming
+One App can be configured via Environment Variables:
 
-[☝️ Return To Top](#environment-variables)
\ No newline at end of file
+**Contents**
+* [`HOLOCRON_MODULE_MAP_URL`](#holocron_module_map_url)
+* [`HOLOCRON_SERVER_MAX_MODULES_RETRY`](#holocron_server_max_modules_retry)
+* [`HOLOCRON_SERVER_MAX_SIM_MODULES_FETCH`](#holocron_server_max_sim_modules_fetch)
+* [`HTTPS_PORT`](#https_port)
+* [`HTTPS_PRIVATE_KEY_PASS_FILE_PATH`](#https_private_key_pass_file_path)
+* [`HTTPS_PRIVATE_KEY_PATH`](#https_private_key_path)
+* [`HTTPS_PUBLIC_CERT_CHAIN_PATH`](#https_public_cert_chain_path)
+* [`HTTPS_TRUSTED_CA_PATH`](#https_trusted_ca_path)
+* [`HTTP_METRICS_PORT`](#http_metrics_port)
+* [`HTTP_ONE_APP_DEV_CDN_PORT`](#http_one_app_dev_cdn_port)
+* [`HTTP_ONE_APP_DEV_PROXY_SERVER_PORT`](#http_one_app_dev_proxy_server_port)
+* [`HTTP_PORT`](#http_port)
+* [`IP_ADDRESS`](#ip_address)
+* [`NODE_ENV`](#node_env)
```

Comment:
Should we order these by most likely to be used? For example `NODE_ENV` would go at the top since it always has to be set. Or maybe group by development only env vars vs production ones vs ones applicable both in dev and prod?

Reply:
I created a By Topic ToC in b02bcb1